sw/airborne/math/pprz_algebra_int.h

```diff
@@ -665,7 +665,9 @@ static inline void int32_vect_smul(int32_t *o, const int32_t *a, const int32_t s
   for (i = 0; i < n; i++) { o[i] = a[i] * s; }
 }
 
-/** Finds a value s in an array and returns the first location at which it is found, else -1 */
+/** Find value s in array a. Returns 1 if found, 0 if not found.
```

Comment:
hahaha, not to nit pic here but if it is a bool shouldn't the return be true or false?